typescript/graphql-pothos/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "typescript-graphql",
```

Comment:
```suggestion
  "name": "typescript-graphql-photos",
```

Reply:
I think there's just a typo in the suggestion - did you mean `typescript-graphql-pothos`?

Reply:
Something felt off when I type that, but I could not figure it out. Thanks for tell me - fixed above in the suggestion. Now it makes sense.